packages/node_modules/glimmer-syntax/lib/parser/handlebars-node-visitors.ts

```diff
@@ -107,7 +107,11 @@ export default {
       changeLines = leadingNewlineDifference(content.original, content.value);
     }
 
-    this.tokenizer.line = this.tokenizer.line + changeLines;
+    if (changeLines) {
```

Comment:
Can this be moved into the previous conditional? It seems like `changeLines` is only set to a truthy value in there so it looks like the `content.rightStripped` check is enough to know whether to apply this change and reset `.column` to 0.

It weirdly appears that changeLines was created before this commit but not used :eyes: